source/hands_on/a2c.rst

```diff
@@ -94,7 +94,40 @@ The policy gradient and value update of A2C is implemented as follows:
+|                     |                 |                                                     |github.com/opendilab/     |  Sb3(17)             |
+|                     |                 |                                                     |DI-engine/tree/main/dizoo/|                      |
+|Pong                 |  20             |.. image:: images/benchmark/pong_a2c.png             |atari/config/serial/      |                      |
+|                     |                 |                                                     |pong/pong_a2c_config      |                      |
+|(PongNoFrameskip-v4) |                 |                                                     |.py>`_                    |                      |
++---------------------+-----------------+-----------------------------------------------------+--------------------------+----------------------+
+|                     |                 |                                                     |`config_link_q <https://  |                      |
+|                     |                 |                                                     |github.com/opendilab/     |  Sb3(3882)           |
+|Qbert                |                 |                                                     |DI-engine/tree/main/dizoo/|  Rllib(3620)         |
+|                     |  4819           |.. image:: images/benchmark/qbert_a2c.png            |atari/config/serial/      |                      |
+|(QbertNoFrameskip-v4)|                 |                                                     |qbert/qbert_a2c_config    |                      |
+|                     |                 |                                                     |.py>`_                    |                      |
++---------------------+-----------------+-----------------------------------------------------+--------------------------+----------------------+
+|                     |                 |                                                     |`config_link_s <https://  |  Sb3(627)            |
+|                     |                 |                                                     |github.com/opendilab/     |  Rllib(692)          |
+|SpaceInvaders        |                 |                                                     |DI-engine/tree/main/dizoo/|                      |
+|                     |  826            |.. image:: images/benchmark/spaceinvaders_a2c.png    |atari/config/serial/      |                      |
+|(SpaceInvadersNoFrame|                 |                                                     |spaceinvaders/space       |                      |
+|skip-v4)             |                 |                                                     |invaders_a2c_config.py>`_ |                      |
++---------------------+-----------------+-----------------------------------------------------+--------------------------+----------------------+
+
```

Comment:
所有表格内的内容居中对齐可能好点

Reply:
**Overview**
A3C(Asynchronous advantage actor-critic) or A2C(advantage actor-critic) algorithm is simple and lightweight framework for deep reinforcement learning that uses asynchronous gradient descent for optimization of deep neural network controllers, where the policy gradient algorithm is combined with an advantage function to reduce variance.

The motivation of A3C/A2C is as follows.
In online on-policy RL, the sequence of observed data is non-stationary and strongly correlated, and is not stable for training.
While in online off-policy RL, using replay buffer and aggregating over memory reduces non-stationarity and decorrelates updates, but has with several drawbacks.
Experience replay uses more memory and computation per real interaction. And in off-policy learning algorithms, data is generated by a somewhat older policy.

In A3C/A2C algorithm, experience replay is replaced with asynchronously execute multiple agents in parallel, on multiple instances of the environment.
This data collection parallelism paradigm decorrelates the agents’ data into a more stationary process, and has stabilizing effect on training, since at any given time-step the parallel agents will be experiencing a variety of different states.  

Therefore, A3C/A2C algorithm and its parallelism paradigm enables a much larger spectrum of on-policy RL algorithms, such as Sarsa, as well as off-policy RL algorithms such as Q-learning, to be applied robustly and effectively.


**Key Equations or Key Graphs**
It is better to add " A(...) =  ... " as the explanation of n-step advantage function.


**Pseudo-code**
It is better to use consistent notions as theta and theta_v as in the section "Key Equations or Key Graphs".
In step 5, the actor network is updated. So, in step 6, should we update theta_v of the critic network?

Reply:
Thank you so much for the comments!

Regrading the comments from Ouwen, 
In the overview part, as we only write the algorithm A2C, I am wondering if it is appropriate to write a long passage to introduce A3C? How about the following?
'A3C(Asynchronous advantage actor-critic) algorithm is simple and lightweight framework for deep reinforcement learning that uses asynchronous gradient descent for optimization of deep neural network controllers. A2C(advantage actor-critic), on the other hand, is the synchronous version of A3C where where the policy gradient algorithm is combined with an advantage function to reduce variance.'

Reply:
Yes, I agree with you. 

All the description is from the origin paper. I hope that will give you some help. It's absolutely fine for you to only select and choose anything you need from the above and add part of it into the doc file. 

It's great for you to make it more concise and to shorten the description about A3C or A2C.